nightmare.vcxproj

```diff
@@ -189,7 +190,7 @@
     <ClInclude Include="src\component\gas_producer_component.h" />
     <ClInclude Include="src\component\light_component.h" />
     <ClInclude Include="src\component\physics_component.h" />
-    <ClInclude Include="src\component\power_component.h" />
+    <ClInclude Include="src\compebonent\power_component.h" />
```

Comment:
This isn't right.

Also, missing filters. The files show up in random places in the project tree.

Reply:
yes, that's bad.
Okay, I'll not update the VS side except with VS.